### DISCUSSION TITLE
Animated Code Snippet Gif

### DISCUSSION THREAD

**Discussion by vikasganiga05** [EddieHubCommunity/support#1175] [Tips and Tricks]
Hi @eddiejaoude and EddieHubCommunity , I found this [Website](https://gif-code-snippet.herokuapp.com/) which creates gif of the code snippets. It's not typing effect line by line code display.

Here is the DEMO
![snippet](https://user-images.githubusercontent.com/74750414/112682869-b9e58f00-8e96-11eb-8fbf-6616422d4e68.gif)

**Comment by eddiejaoude** [comment]
Thank you 👍

**Reply by vikasganiga05** [reply]
Welcome : )
